federatedscope/autotune/algos.py

```diff
@@ -57,7 +57,7 @@ def run(self):
                                server_class=get_server_cls(self._trial_cfg),
                                client_class=get_client_cls(self._trial_cfg),
                                config=self._trial_cfg.clone(),
-                               client_config=self._client_cfgs)
+                               client_configs=self._client_cfgs)
```

Comment:
to avoid such kinds of mistakes, we need to improve the coverage of our unit test cases